pkgs/applications/editors/jupyter-kernels/xeus-cling/xeus-cling.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, callPackage
+, cmake
+, fetchFromGitHub
+, gcc
+, git
+
+, clangStdenv
+, llvmPackages_9
+
```

Comment:
Could you remove the unnecessary whitespace from the code? We need to maintain the same style as other packages, like https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/pp/ppsspp/package.nix#L20

Reply:
Done